src/test/java/com/softensy/SoftensySpringBoot/service/serviceImpl/DoctorServiceImplTest.java

```diff
@@ -102,17 +154,20 @@ void givenDoctor_whenUpdateDoctor_thenReturnDoctorAndVerifyDoctorUpdate() {
     @DisplayName("delete doctor")
```

Comment:
displayName for tests should display understandable test name for a person who did not see your code and functionality!